server/services/phase.ts

```diff
@@ -111,16 +125,22 @@ export const getAllSitePhases = async (siteId: number): Promise<sitePhase[]> =>
   }));
 };
 
-export const getAllPhases = async (query = null) => {
+/**
+ * Gets all phases with or without associated allocations
+ *
+ * @param includeAllocations  condition to include associated allocation records
+ * @returns phase[]   An array of phases with formatted
```

Comment:
Typo in description - with formatted what, exactly?

Reply:
fixed! the sun was shining bright on my screen this morning, tough to see the comments.